grunt/config/less.js

```diff
@@ -1,116 +1,122 @@
-module.exports = function (grunt, options) {
-    var convertSlashes = /\\/g;
-    var path = require("path");
+module.exports = function(grunt, options) {
+  var convertSlashes = /\\/g;
+  var path = require("path");
 
-    function alphanumericOrder(a, b) {
-        return a > b ? 1 : a < b ? -1 : 0;
-    }
+  function alphanumericOrder(a, b) {
+    return a > b ? 1 : a < b ? -1 : 0;
+  }
 
-    function lengthOrder(a, b) {
-        return a.length > b.length ? 1 : a.length < b.length ? -1 : 0;
-    }
+  function lengthOrder(a, b) {
+    return a.length > b.length ? 1 : a.length < b.length ? -1 : 0;
+  }
 
-    function compareFilePaths(a, b) {
-        /**
-         * If not in the same folder sort alphanumerically
-         */
-        var aParsed = path.parse(a);
-        var bParsed = path.parse(b);
-        if (aParsed.dir !== bParsed.dir) return alphanumericOrder(a, b);
+  function compareFilePathsNesting(a, b) {
```

Comment:
this has since been removed

https://github.com/adaptlearning/adapt_framework/commit/82499f0dfc90d9214f5bfcf17b8129ef79c9fc5c